### PR TITLE
dbeaver: update to 23.1.0.

### PR DESCRIPTION
**Pull request by ar-jan**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/dbeaver/files/dbeaver.desktop [hunk 1]
```diff
@@ -1,9 +1,8 @@
 [Desktop Entry]
 Name=DBeaver
 Comment=SQL Integrated Development Environment
-Icon=dbeaver
+Icon=/usr/lib/dbeaver/icon.xpm
```
Comment (hervyqa) [inline srcpkgs/dbeaver/files/dbeaver.desktop:4]: @ar-jan why did you change the icon path?
this will break other icon like `papirus-icons` which already provide default icon from the theme.

Comment (ar-jan) [reply srcpkgs/dbeaver/files/dbeaver.desktop:4]: `Icon=dbeaver` never worked for me so I had no icon for dbeaver (I use KDE5 Plasma). I wasn't aware this change would break other configs, how does `papirus-icons` work?

Comment (hervyqa) [reply srcpkgs/dbeaver/files/dbeaver.desktop:4]: didn't work, `papirus-icon-theme` couldn't read it.
